app/components/sy-timepicker/component.js

```diff
@@ -70,7 +72,15 @@ export default Component.extend({
    * @private
    */
   _add(h, m) {
-    return moment(this.get('value')).add({ h, m })
+    let base = this.get('value')
+
+    if (!base) {
+      base = [h, m].any(n => n < 0)
+        ? this.get('max').add(1, 'minute')
+        : this.get('min')
```

Comment:
I'm not sure if i get this. Why is the base`max + 1min` if `h` or `m` is negative?

Reply:
Oh that's to make sure you're not entering "yesterday" by clicking the down arrow, right?

Reply:
If we don't have a current value and press arrow down, which either subtracts 15 minutes or one hour, we subtract it from the current max which is normally 23:59 if not given. So we need to add a minute to get a valid value.

Reply:
:+1: